tests/automation/test_package.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+import pytest
+from pytest import mark as m
+
+from automation.package import FailedTestError, LibError, Package, PackageError
+from automation_fixtures import test_channel, test_recipebook
+
+_ = test_channel
+_ = test_recipebook
+
+
+@m.describe("Package")
+class TestPackage(object):
+    @m.it("Can equate to an identical Package object")
+    def test_equals(self):
+        assert Package(("success", "1.0.0")) == Package(("success", "1.0.0"))
+        assert Package(("success", "1.0.0")) != Package(("fail", "1.0.0"))
+
+    @m.context("When constructed without a Recipebook")
+    @m.it("Accessing sub-package names raises an error")
+    def test_sub_packages_no_recipebook(self):
+        with pytest.raises(PackageError,
+                           match=r"^Cannot determine sub-packages"):
+            Package(("success", "1.0.0")).sub_packages()
+
+    @m.context("When constructed with a Recipebook")
+    @m.it("Can get sub-package names")
+    def test_sub_packages_recipebook(self, test_recipebook):
+        r = test_recipebook
+
+        with pytest.raises(PackageError,
+                           match=r"^Cannot determine sub-packages"):
+            Package(("success", "1.0.0")).sub_packages()
+
```

Comment:
This is already covered by the previous test

Reply:
I should replace the first one with a package that actually has sub-packages (and then doesn't find them because the recipebook isn't available).

Reply:
Will fix by using a package that actually has sub-packages, in the previous test.